microsite/src/pages/home/hubSpotNewAdoptersForm.module.scss

```diff
@@ -3,7 +3,7 @@
   right: 0;
   display: flex;
   position: fixed;
-  transform: translateY(-50%);
+  transform: translateY(-40%);
```

Comment:
I'm not sure that this a fix for this. This is meant to be half of the size of the element, I think you'll probably want to make sure that the element has a different sizing.